Sources/TuistKit/Utils/ManifestGraphLoader.swift

```diff
@@ -92,7 +92,15 @@ final class ManifestGraphLoader: ManifestGraphLoading {
         let plugins = try await loadPlugins(at: path)
 
         // Load DependenciesGraph
-        let dependenciesGraph = try dependenciesGraphController.load(at: path)
+        let dependenciesGraph: TuistGraph.DependenciesGraph
+
+        // We load the denpendencies manifests since its allowed to have a `Dependencies.swift`
+        // and not have any dependencies. Check `plugins.feature` acceptance test.
+        if let dependencies = try? manifestLoader.loadDependencies(at: path), dependencies.hasDependencies {
+            dependenciesGraph = try dependenciesGraphController.load(at: path)
+        } else {
+            dependenciesGraph = .none
+        }
```

Comment:
@danyf90 had to add this since one of the acceptance tests fail if we have a `dependencies.swift` file where the file has a dependencies model that has no [external dependencies](https://github.com/tuist/tuist/blob/main/projects/tuist/fixtures/app_with_plugins/Tuist/Dependencies.swift). And thus we have to load the manifest first and check if it has external dependencies

Reply:
Will need to provide a better message there though. thats just a place holder

Reply:
I would just update the acceptance test sample project to remove the empty Dependencies.swift

Reply:
Makes sense, then I would just write an empty graph.json instead of doing `clean` in this case:
```
    public func save(
        dependenciesGraph: TuistGraph.DependenciesGraph,
        to path: AbsolutePath
    ) throws {
        if dependenciesGraph.externalDependencies.isEmpty {
            try dependenciesGraphController.clean(at: path)
        } else {
            try dependenciesGraphController.save(dependenciesGraph, to: path)
        }
    }

```

Reply:
Perfect! then!